src/content/en/2020/resource-hints.md

```diff
@@ -231,47 +238,47 @@ Implementing a `dns-prefetch` fallback in the same `<link>` tag causes a [bug](h
-  caption="Pages that pass the preload Lighthouse audit",
+  caption="Pages that pass the `preload` Lighthouse audit.",
   content="84.6%",
   classes="big-number",
   sheets_gid="1047875076",
   sql_file="lighthouse_preload.sql"
 ) }}
 
-Running Lighthouse’s "[Preload key requests](https://web.dev/uses-rel-preload/)" audit resulted in 84.6% of pages passing the test, which is an astonishing result. If you are looking to use `preload` for the first time, remember, fonts and critical scripts are a good place to start.
+Running Lighthouse's "[Preload key requests](https://web.dev/uses-rel-preload/)" audit resulted in 84.6% of pages passing the test, which is an astonishing result. If you are looking to use `preload` for the first time, remember, fonts and critical scripts are a good place to start.
 
-{# TODO - revisit this sentence - Ref https://github.com/HTTPArchive/almanac.httparchive.org/pull/1587#discussion_r532291496 #}
+{# TODO(authors) - revisit this sentence - Ref https://github.com/HTTPArchive/almanac.httparchive.org/pull/1587#discussion_r532291496 #}
 
 ### Native Lazy Loading
 
-Now let’s celebrate the first year of the [Native Lazy Loading](https://addyosmani.com/blog/lazy-loading/) API, which at the time of publishing already has over [72%](https://caniuse.com/loading-lazy-attr) browser support. This new API can be used to defer the load of below-the-fold iframes and images on the page until the user scrolls near them. This can reduce data usage, memory usage, and helps speed up above-the-fold content. Opting-in to lazy load is as simple as adding `loading=lazy`  on `<iframe>` or `<img>` elements.
+Now let's celebrate the first year of the [Native Lazy Loading](https://addyosmani.com/blog/lazy-loading/) API, which at the time of publishing already has over [72%](https://caniuse.com/loading-lazy-attr) browser support. This new API can be used to defer the load of below-the-fold iframes and images on the page until the user scrolls near them. This can reduce data usage, memory usage, and helps speed up above-the-fold content. Opting-in to lazy load is as simple as adding `loading=lazy`  on `<iframe>` or `<img>` elements.
 
-{# TODO - revisit this sentence - Ref https://github.com/HTTPArchive/almanac.httparchive.org/pull/1587#discussion_r533106799 #}
+{# TODO(authors) - revisit this sentence - Ref https://github.com/HTTPArchive/almanac.httparchive.org/pull/1587#discussion_r533106799 #}
 
 {{ figure_markup(
-  caption="Percentage of pages using native lazy loading set to “lazy”",
+  caption="Percentage of pages using native lazy loading.",
   content="3.87%",
   classes="big-number",
   sheets_gid="2039808014",
   sql_file="native_lazy_loading_attrs.sql"
 ) }}
 
-{# TODO - revisit this figure - Ref https://github.com/HTTPArchive/almanac.httparchive.org/pull/1587#discussion_r532292106 #}
+{# TODO(authors) - revisit this figure - Ref https://github.com/HTTPArchive/almanac.httparchive.org/pull/1587#discussion_r532292106 #}
```

Comment:
@Zizzamia

Reply:
Not quite sure what's the split between images vs iframe based on [native_lazy_loading_attrs.sql](
https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/sql/2020/21_Resource_Hints/native_lazy_loading_attrs.sql).

@khempenius can you helps us clarify more about this data, @bazzadp was asking about it here https://github.com/HTTPArchive/almanac.httparchive.org/pull/1587